components/filter/filter.js

```diff
@@ -750,6 +750,8 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 				this._activeFiltersSubscribers.updateSubscribers();
 			} else if (prop === 'loading') {
 				shouldResizeDropdown = true;
+			} else if (prop === 'text') {
+				this._activeFiltersSubscribers.updateSubscribers();
```

Comment:
The `text` on a `filter-tag` previously wasn't updating if it changed, so this updates that. This potentially normally wouldn't be an issue, however with the new filter date value components the lang terms were sometimes later to come in than when `text` was actually set on the filter-tags so this keeps that in sync.